packages/fgtclb/academic-jobs/Resources/Private/Partials/Job/Forms/Checkbox.html

```diff
@@ -0,0 +1,38 @@
+<html
+    lang="en"
+    xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
+    data-namespace-typo3-fluid="true"
+>
+
+<f:variable name="elementId" value="job.{element.identifier}"/>
+
+<f:variable name="required" value="{element.required}" />
+
+<f:for each="{element.validations.{element.identifier}}" as="validation">
```

Comment:
Really ? We are doing that in a f****** template instead of preparing the element data correctly upfront (controller) ?!?